source/jormungandr/jormungandr/interfaces/v1/fields.py

```diff
@@ -429,11 +429,12 @@ def format(self, value):
 
 class Co2Emission(fields.Raw):
```

Comment:
this class is still used by co2_emission in journeys, i'll see if i can remove it